.github/workflows/build-publish-pr.yml

```diff
@@ -13,6 +13,7 @@ jobs:
   build-publish-untrusted:
     if: ${{ ! startsWith(github.ref_name, 'release/') }}
     runs-on: ubuntu-20.04
+    environment: sdlc
```

Comment:
Above comment (line 5) has "# It doesn't use an environment, has its own special IAM role, does not sign".

Reply:
Nice, updated in 410a7d470fb3ed4e2db670cb1493a3b838a21a23